ansys/mapdl/core/errors.py

```diff
@@ -70,6 +75,25 @@ def __init__(self, msg=LOCKFILE_MSG):
         RuntimeError.__init__(self, msg)
 
 
+class LicenseServerConnectionError(RuntimeError):
+    """Error when the license server is not available."""
+    def __init__(self, error_message='', head_message='', tail_message='', **kwargs):
```

Comment:
```suggestion
    def __init__(self, error_message=None, head_message=None, tail_message=None, **kwargs):
```
This isn't very different from what you've done, but I think it is more common to set an optional argument to None. In that case the following checks would be different as well:

```
        if head_message is not None:
            msg = msg + '\n' + head_message + '\n'

        if error_message is not None:
            msg =  msg + '\nError recorded:' '\n***\n' + error_message + '\n***\n'

        if tail_message is not None:
            msg = msg + '\n' + tail_message 
```

Reply:
Ok. Done!